clang/test/Driver/sycl-instrumentation-old-model.c

```diff
@@ -20,19 +20,25 @@
 // CHECK-SPIRV-SAME: "{{.*}}libsycl-itt-stubs.bc"
 // CHECK-HOST-NOT: "-cc1"{{.*}} "-fsycl-is-host"{{.*}} "-fsycl-instrument-device-code"
 
-// ITT annotations in device code are disabled by default. However, for SYCL offloading,
-// we still link ITT annotations libraries to ensure ABI compatibility with previous release.
-// RUN: %clangxx -fsycl --no-offload-new-driver -fsycl-targets=spir64 -### %s 2>&1 \
+// ITT annotations in device code are disabled by default. However, for SYCL
+// offloading, we still link ITT annotations libraries to ensure ABI
+// compatibility with previous release.
+// RUN: %clangxx -fsycl --no-offload-new-driver -fsycl-targets=spir64 -### \
+// RUN:   --sysroot=%S/Inputs/SYCL %s 2>&1 \
 // RUN: | FileCheck -check-prefixes=CHECK-ITT-LINK-ONLY %s
-// RUN: %clangxx -fsycl --no-offload-new-driver -fsycl-targets=nvptx64-nvidia-cuda -nocudalib -### %s 2>&1 \
+// RUN: %clangxx -fsycl --no-offload-new-driver --sysroot=%S/Inputs/SYCL \
+// RUN:   -fsycl-targets=nvptx64-nvidia-cuda -nocudalib -### %s 2>&1 \
 // RUN: | FileCheck -check-prefixes=CHECK-NONPASSED %s
 
 // CHECK-ITT-LINK-ONLY-NOT: "-fsycl-instrument-device-code"
 // CHECK-ITT-LINK-ONLY: llvm-link{{.*}} {{.*}}libsycl-itt-{{.*}}
 
-// RUN: %clangxx -fsycl --no-offload-new-driver -fno-sycl-instrument-device-code -fsycl-targets=spir64 -### %s 2>&1 \
+// RUN: %clangxx -fsycl --no-offload-new-driver --sysroot=%S/Inputs/SYCL \
```

Comment:
A comment describing these invocations would be helpful.
Also, does this change apply to `clang_cl` invocation as well?

Reply:
For now, no.  There is a FIXME at the to of the file:
// FIXME: Force linux targets to allow for the libraries to be found.  Dummy
// inputs for --sysroot should be updated to work better for Windows.

I'll add a comment

Reply:
Does this apply when passing `--offload-new-driver` ?

Reply:
This test is for the old model specifically.  But checking with `--offload-new-driver` the output does not look impacted by the existence of the libraries.